src/components/TokenOverlay/index.tsx

```diff
@@ -1,29 +1,30 @@
 import React, { Component } from 'react'
-import TokenItem from '../TokenItem'
-
-// TODO: move tokens and codes to global config or redux
-const code2tokenMap = {
-  ETH: 'ETHER',
-  GNO: 'GNOSIS',
-  REP: 'AUGUR',
-  '1ST': 'FIRST BLOOD',
-  OMG: 'OMISEGO',
-  GNT: 'GOLEM',
-}
+import TokenOverlayHeader from '../TokenOverlayHeader'
+import TokenList from '../TokenList'
+import { code2tokenMap, TokenCode } from 'globals'
+import { createSelector } from 'reselect'
+
+const filterTokens = createSelector(
+  (state: TokenOverlayState, _: TokenOverlayProps) => state.filter.toUpperCase(),
+  (_: TokenOverlayState, props: TokenOverlayProps) => props.tokenCodeList,
```

Comment:
what is the '_' used as param 1?

Reply:
Just a convention meaning `_` param isn't going to be used. The first two arrow function must have the same signature as that is what `filterTokens` will receive as input and pass onto them.